src/components/Form.js

```diff
@@ -74,42 +105,46 @@ export default class Form extends Component {
     return null;
   }
 
-  onChange = (formData, options={validate: false}) => {
+  onChange = (formData, options = {validate: false}) => {
```

Comment:
Sorry, I can't find suitable rule :disappointed:
And if possible, I would suggest that changes in the rules out of scope of this PR